enterprise/backend/src/metabase_enterprise/sso/api/interface.clj

```diff
@@ -3,11 +3,18 @@
    [metabase-enterprise.sso.integrations.sso-settings :as sso-settings]
    [metabase.util.i18n :refer [tru]]))
 
+(defn- jwt-saml-both-enabled
```

Comment:
nit: this function name sounds to me like it returns a boolean; maybe just call it something like `select-sso-backend`?

you could keep `jwt-saml-both-enabled` and have it just abstract `(and (sso-settings/saml-enabled) (sso-settings/jwt-enabled))` maybe

Reply:
Yea makes sense, I was thinking maybe sso didn't make sense since we are selecting only across SAML and JWT, but you're right the convention here is like sso -> saml,jwt.